### PR TITLE
Update SDK and libraries

### PR DESCRIPTION
**Pull request by iamareebjamal**
**Summary of changes**

Updated SDK version and libraries

Please make sure these boxes are checked before submitting your pull request - thanks! [Guide](./PULL_REQUEST_GUIDE.md)

- [x] Code is formatted.

- [x] Run the lint tests with `./gradlew lintDebug` and make sure BUILD is SUCCESSFUL

- [x] If new feature is implemeted then it is compatible with Night theme too.

- [x] Commit messages are well-written.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@iamareebjamal, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/).

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*
				
@iamareebjamal, thanks for signing the CLA!

**Comment by iamareebjamal** [comment]
Fixes #452

**Comment by iamareebjamal** [comment]
Please review @kunall17 @saketkumar95

File: app/build.gradle [hunk 1]
```diff
@@ -67,25 +67,25 @@ android {
...
-    compile 'com.android.support:appcompat-v7:24.1.1'
-    compile 'com.android.support:design:24.1.1'
-    compile 'com.android.support:recyclerview-v7:24.1.1'
+    compile 'com.android.support:appcompat-v7:25.1.1'
+    compile 'com.android.support:design:25.1.1'
+    compile 'com.android.support:recyclerview-v7:25.1.1'
     compile 'com.google.android.gms:play-services-gcm:8.4.0'
     compile 'com.google.android.gms:play-services-auth:8.4.0'
     compile 'com.squareup.retrofit2:retrofit:2.2.0'
-    compile 'com.squareup.okhttp3:logging-interceptor:3+'
```
Comment (Sam1301) [inline app/build.gradle]: please let the version numbers be in '+' form as whenever a newer version is available, travis fails due to NewerVersionAvailable constraint.

File: app/build.gradle [hunk 2]
```diff
@@ -67,25 +67,25 @@ android {
...
     compile 'com.squareup.retrofit2:converter-gson:2.2.0'
-    compile 'com.squareup.okhttp3:okhttp:3+'
```
Comment (Sam1301) [inline app/build.gradle]: same as above

File: app/build.gradle [hunk 3]
```diff
@@ -67,25 +67,25 @@ android {
...
     compile 'com.j256.ormlite:ormlite-android:5.0'
     compile 'commons-lang:commons-lang:2.6'
     compile 'org.ccil.cowan.tagsoup:tagsoup:1.2.1'
     compile 'com.squareup.picasso:picasso:2.5.2'
     //noinspection GradleCompatible
-    compile 'com.android.support:customtabs:23.3.0'
+    compile 'com.android.support:customtabs:25.1.1'
     compile 'com.github.bumptech.glide:glide:3.7.0'
-    compile 'com.android.support:support-v4:24.1.0'
-    compile 'com.theartofdev.edmodo:android-image-cropper:2.3.+'
```
Comment (Sam1301) [inline app/build.gradle]: same as above

File: app/build.gradle [hunk 4]
```diff
@@ -4,7 +4,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'io.fabric.tools:gradle:1.+'
```
Comment (Sam1301) [inline app/build.gradle]: same as above

File: app/build.gradle (hunk 3 above)
Comment (iamareebjamal) [inline app/build.gradle]: OK

**Comment by iamareebjamal** [comment]
Done @Sam1301

**Comment by iamareebjamal** [comment]
Done @Sam1301

**Comment by brainwane** [comment]
Thanks for writing a descriptive commit message. [Here's a guide to help you write commit messages](https://zulip.readthedocs.io/en/latest/version-control.html#commit-messages) that are more consistent with our style, which makes it easier for us in the future -- when someone's trying to solve a new problem, if commit messages are all in the same format, it's easier to skim them in logs and quickly understand what's happening. For instance, having a separate line at the end of the commit message that includes the issue number in the format we specify will help us use GitHub to automatically close that issue once we merge your commit.

[Here are some tips on tidying your commit history, including changing your commit messages.](https://zulip.readthedocs.io/en/latest/git-guide.html#examine-and-tidy-your-commit-history) And when you update your code or your commit message, you can [update your pull request](https://zulip.readthedocs.io/en/latest/git-guide.html#update-a-pull-request) and the changes will appear here. :)

Thanks @iamareebjamal !

**Comment by iamareebjamal** [comment]
Sorry for late response, I have mid terms going on @brainwane

**Comment by brainwane** [comment]
@iamareebjamal Thanks for the update! You ought to take out the parentheses in the final line of the commit message and add a period at the end, though.

**Comment by iamareebjamal** [comment]
Done @brainwane

**Comment by Sam1301** [comment]
Merged, thanks @iamareebjamal!

**Pull request closed without merging**